core/services/ping/ping360_driver.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from shutil import which
+from subprocess import Popen
+from typing import Optional
+
+import serial
+
+from pingdriver import PingDriver
+from pingutils import PingDeviceDescriptor
+
+
+class Ping360Driver(PingDriver):
+    def __init__(self, ping: PingDeviceDescriptor, port: int) -> None:
+        super().__init__(ping, port)
+        self.process: Optional[Popen[bytes]] = None
+        self.baudrate: int = 2000000
```

Comment:
This may not work in all scenarios

Reply:
you mean 2M for baudrate? as in "some setups may fail to communicate with the ping360 at 2M"?

I can add some logic to raise the baudrate progressively until talking to the sensor fails, then go one step back. Thoughts?

Reply:
Yep, we could raise or decrease the baudrate, your choice, I just recommend to do N times to make sure that is a good baudrate, we could also have a dynamic baudrate change while using the sensor and not changing only when starting the connection, but that's up to you.

Reply:
I think that you should start at the lowest baudrate, if it does not communicate we could just ignore it and start with the next candidate.